packages/app-config/scripts/package-subtree.sh

```
@@ -487,24 +554,157 @@ apply_patch_based_update() {
+              failed_file="${failed_file#$WORKSPACE_LOCATION/$TARGET_RELATIVE/}"
+              failed_files+=("$failed_file")
+            fi
+          done <<< "$apply_error"
+          
+          # If patch was completely rejected, try to apply files individually
+          if [ "$has_git_conflicts" = false ] && [ ${#failed_files[@]} -gt 0 ]; then
+            info_msg "Attempting to apply other files from the patch individually..."
+            echo ""
+            
+            # Extract all files from the patch
+            local all_files=$(grep '^diff --git' "$filtered_patch" | sed 's/^diff --git a\/.* b\///' || true)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)